apps/modernization-api/src/main/java/gov/cdc/nbs/graphql/filter/PatientFilter.java

```diff
@@ -21,6 +21,11 @@
 @EqualsAndHashCode
 @JsonInclude(Include.NON_NULL)
 public class PatientFilter {
+    public PatientFilter(RecordStatus... recordStatus) {
```

Comment:
This can still allow zero `recordStatus`.  it's tricky to enforce this with jackson.  Maybe something like
```java
public PatientFilter(RecordStatus required, RecordStatus...rest) {
...
}

public PatientFilter() {
  this(RecordStatus.ACTIVE);
}
```

This would introduce a default of `ACTIVE` though.

Reply:
I think a default of `ACTIVE` is fine. Updated to your recommendation